extensions/wikia/GlobalWatchlist/GlobalWatchlist.i18n.php

```diff
@@ -0,0 +1,425 @@
+<?php
+
+$messages = array();
+
+$messages['en'] = array(
+	"tog-watchlistdigest" => "E-mail me a Weekly Watchlist Digest",
+	"tog-watchlistdigestclear" => "Mark all pages from my Weekly Watchlist Digest as visited",
```

Comment:
As part of SOC-530 I took out this entire file, but realized that we still need these 2 messages which are part of Special:Preferences for the weekly digest.